code/modules/fishing/fishing_rod.dm

```diff
@@ -0,0 +1,117 @@
+
+/obj/item/weapon/material/fishing_rod
+	name = "crude fishing rod"
+	desc = "A crude rod made for catching fish."
+	icon_state = "fishing_rod"
+	item_state = "fishing_rod"
+	force_divisor = 0.25
+	throwforce = 7
+	sharp = TRUE
+	attack_verb = list("whipped", "battered", "slapped", "fished", "hooked")
+	hitsound = 'sound/weapons/punchmiss.ogg'
+	applies_material_colour = TRUE
+	default_material = "wood"
+	can_dull = FALSE
+	var/strung = TRUE
+	var/line_break = TRUE
+
+	var/obj/item/weapon/reagent_containers/food/snacks/Bait
+	var/bait_type = /obj/item/weapon/reagent_containers/food/snacks
+
+	var/cast = FALSE
+
+	attackspeed = 3 SECONDS
+
+/obj/item/weapon/material/fishing_rod/built
+	strung = FALSE
+
+/obj/item/weapon/material/fishing_rod/examine(mob/M as mob)
+	..()
+	if(Bait)
+		to_chat(M, "<span class='notice'>\The [src] has \the [Bait] hanging on its hook.</span>")
+		Bait.examine(M)
+
+/obj/item/weapon/material/fishing_rod/CtrlClick(mob/user)
+	if((src.loc == user || Adjacent(user)) && Bait)
+		Bait.forceMove(get_turf(user))
+		to_chat(user, "<span class='notice'>You remove the bait from \the [src].</span>")
+		Bait = null
+	else
+		..()
+
+/obj/item/weapon/material/fishing_rod/Initialize()
+	..()
+	update_icon()
+
+/obj/item/weapon/material/fishing_rod/attackby(obj/item/I as obj, mob/user as mob)
+	if(I.is_wirecutter() && strung)
+		strung = FALSE
+		to_chat(user, "<span class='notice'>You cut \the [src]'s string!</span>")
+		update_icon()
+		return
+	else if(istype(I, /obj/item/stack/cable_coil) && !strung)
+		var/obj/item/stack/cable_coil/C = I
+		if(C.amount < 5)
+			to_chat(user, "<span class='warning'>You do not have enough length in \the [C] to string this!</span>")
+			return
+		if(do_after(user, rand(10 SECONDS, 20 SECONDS)))
+			C.use(5)
+			strung = TRUE
+			to_chat(user, "<span class='notice'>You re-string \the [src]!</span>")
```

Comment:
`You string \the [src]!` Don't know if the user strung the rod in the first place

Reply:
Fixt